.github/workflows/ci.yml

```diff
@@ -31,14 +31,14 @@ jobs:
       env:
         CI: true
     - run: npm run lint
-      if: matrix.node-version == '12.x'
+      if: matrix.node-version == '16.x'
     - run: npm run docs:diff
-      if: matrix.node-version == '12.x'
+      if: matrix.node-version == '16.x'
     - run: npm run bundlewatch
-      if: matrix.node-version == '12.x'
+      if: matrix.node-version == '16.x'
       env:
         BUNDLEWATCH_GITHUB_TOKEN: ${{ secrets.BUNDLEWATCH_GITHUB_TOKEN }}
     - run: npm run test:node
-      if: matrix.node-version == '12.x' || matrix.node-version == '14.x'
+      if: matrix.node-version == '16.x' || matrix.node-version == '14.x'
     - run: npm run test:pack
-      if: matrix.node-version == '14.x'
+      if: matrix.node-version == '16.x' || matrix.node-version == '14.x'
```

Comment:
I think that these two tests should run on `>= 12`? That is, all that supports ESM

Reply:
Early versions of 12.x didn't support ESM IIRC… That's why historically we didn't test there.

But I guess nowadays it's safe to run these tests on Node 12+, so thanks for the catch!